build.sbt

```diff
@@ -87,7 +87,8 @@ lazy val orchardWS = (project in file("orchard-ws")).
       "com.fasterxml.jackson.core" % "jackson-databind" % "2.15.3",
       "com.fasterxml.jackson.core" % "jackson-core" % "2.15.3",
       "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.15.3",
-      "com.fasterxml.jackson.dataformat" %% "jackson-dataformat-cbor" % "2.17.0"
+      "com.fasterxml.jackson.dataformat" %% "jackson-dataformat-cbor" % "2.17.0",
+      "io.netty" % "netty-handler" % "5.0.0.Alpha1"
```

Comment:
alpha? 🤔

Reply:
would update to play 3.0.5 solve the issue?

Reply:
@realstraw I had the same dilema on using alpha version. That is why I removed in my previous commit but returned it back to have conversation on it.

Reply:
I will remove this netty upgrade from this PR to have the fix with play upgrade